src/rules/noUnsafeQuery.ts

```diff
@@ -43,9 +48,9 @@ const create = (context) => {
 
       const legacyTagName = tag?.name.toLowerCase();
```

Comment:
Also I think lowercasing is unclear there because /format rule hasn't same behaviour